server/api/game/constants.js

```diff
@@ -28,7 +28,10 @@ export const getBoard = () => [
 
 export const boardTranspose = getBoard()[0].map((col, i) => getBoard().map(row => row[i]));
 
-export const refreshRate = 700;
+
+const framesPerSecond = 3;
```

Comment:
There are some functions which are dependent on the Refresh Rate. Those functions were changed as well?

Reply:
There was only 1 function using `refreshRate` and that has been handled in this PR.